cluster-operator/src/main/resources/cluster-roles/020-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -102,6 +102,7 @@ rules:
   - deployments/scale
   - deployments/status
   - statefulsets
+  - replicasets
```

Comment:
Out of curiosity, why we need replicasets here which wasn't there before?

Reply:
I think this is related to the Fabric8 upgrade - it just never jumped out because we never deleted a deployment (Kafka Connect deployment is now deleted by Garbage collection for some time). The client simply tries to delete the replica sets when deleting the deployment.